src/FffGcodeWriter.cpp

```diff
@@ -2379,159 +2380,187 @@ bool FffGcodeWriter::addSupportToGCode(const SliceDataStorage& storage, LayerPla
     return support_added;
 }
 
-
-bool FffGcodeWriter::processSupportInfill(const SliceDataStorage& storage, LayerPlan& gcode_layer) const
+bool FffGcodeWriter::processSupport(const SliceDataStorage& storage, LayerPlan& gcode_layer) const
```

Comment:
This function doesn't process the support roof or support bottom, which is also part of support. So I think this rename is confusing!